### ISSUE TITLE
navbar - angular filters don't work on Android

### ISSUE THREAD

**Issue by akatasonov** [AppGyver/steroids#905]
I'm using angular-translate and it provides translation via the “translate” filter. So, my navbar code looks like this
```
<super-navbar>
  <super-navbar-title>
    {{ 'APP_NAME' | translate }}
  </super-navbar-title>
</super-navbar>
```
It all works fine in iOS (emulator and iPhone 5s) – i.e. the translated title is shown. However, on Android (5.0.2, Google Nexus 7 and also multiple other devices) the navbar title is displayed as {{ 'APP_NAME' | translate }}. So it seems the navbar isn't updated once angular filters change the title value. Are there any work arounds?

**Comment by harsaharsa** [comment]
If you have the double-mustache there, then Angular is not being loaded correctly at all. See what's going on in the WebView via Chrome Remote Inspector – are there any errors?

**Comment by akatasonov** [comment]
I do have double mustache there. Screen shots of the app screen and Chrome Remote Inspector are attached.

![image](https://cloud.githubusercontent.com/assets/6917547/7229965/1d533d94-e772-11e4-8919-f21ce173f273.png)


![image](https://cloud.githubusercontent.com/assets/6917547/7229903/8485d37e-e771-11e4-8d12-d0bb7a65fb9e.png)

**Comment by andybbtt11** [comment]
I ran into the same exact issue today.

**Comment by harsaharsa** [comment]
Can you add `ng-bind=navbar-title` into the `super-navbar` and then update `$scope.navbar-title` variable instead?

**Comment by andybbtt11** [comment]
Yeah, @harsaharsa is right, using ng-bind="item" instead of {{item}} works for me.

**Comment by akatasonov** [comment]
@harsaharsa Your solution works, thanks!!
